sync/class.jetpack-sync-queue.php

```diff
@@ -66,14 +81,21 @@ function add_all( $items ) {
 		$rows = array();
 
 		for ( $i = 0; $i < count( $items ); $i += 1 ) {
+			if ( apply_filters( 'jetpack_skip_sync_item', false, $items[ $i ]) ) {
```

Comment:
Could you add a small docblock here, like `/** This action is already documented in sync/class.jetpack-sync-queue.php */`?